gr-digital/lib/constellation.cc

```diff
@@ -131,7 +132,7 @@ float constellation::get_distance(unsigned int index, const gr_complex* sample)
 
 unsigned int constellation::get_closest_point(const gr_complex* sample)
 {
-    unsigned int min_index = 0;
+    unsigned int min_index;
     float min_euclid_dist;
     float euclid_dist;
```

Comment:
Now that you touched this function, I had a look at it.

https://github.com/gnuradio/gnuradio/blob/7a42de461b4d312fb736abbeeaacd1388a592704/gr-digital/lib/constellation.cc#L133-L149

I suggest to combine declaration and initialization. at least for `min_index` and `min_euclid_dist`. These should be the next 2 lines (L139-140) anyways.

In case of `euclid_dist`, I'm tempted to suggest to move its declaration to line L142. It is more precise to have its declaration there because the variable is only used in the for loop. I doubt this has any negative performance implications.